satellite/satellitedb/database.go

```diff
@@ -59,47 +57,23 @@ func NewInMemory() (satellite.DB, error) {
 	return New("sqlite3://file::memory:?mode=memory")
 }
 
-// DROP_ALL_TABLES drops all tables in the database.
-// Deprected: do not use in production.
-func DROP_ALL_TABLES(dbAny satellite.DB) (err error) { //nolint: ignore all caps requirement for dangerous function
-	var db *DB
-	switch temp := dbAny.(type) {
-	case *DB:
-		db = temp
-	case *locked:
-		return DROP_ALL_TABLES(temp.db)
-	default:
-		return errors.New("unsupported implementation")
+// SetSchema sets the schema
+func (db *DB) SetSchema(schema string) error {
+	switch db.driver {
+	case "postgres":
+		_, err := db.db.Exec("create schema ?; set search_path to ?;", schema, schema)
```

Comment:
Nope, it doesn't. :D typos.

It creates a new connection for each tests, so should be fine.

Reply:
@thepaul thoughts on this fix?

Reply:
since the schema is guaranteed to be a valid pg identifier (as things are now) i think this is fine.

if you did want to quote it, you could just import "github.com/lib/pq" and use `pq.QuoteIdentifier(schema)`.